docs/source/install/kernel_install.rst

```diff
@@ -53,3 +53,28 @@ For example, using conda environments:
 The ``--name`` value is used by Jupyter internally. These commands will overwrite
 any existing kernel with the same name. ``--display-name`` is what you see in
 the notebook menus.
+
+Using virtualenv or conda envs, you can make your IPython kernel in one env available to Jupyter in a different env. To do so, run ipykernel install from the kernel's env, with --prefix pointing to the Jupyter env:
+
+.. sourcecode:: bash
+
+    /path/to/kernel/env/bin/python -m ipykernel install --prefix=/path/to/jupyter/env --name 'python-my-env'
+
+Note that this command will create new configuration for kernel in one of it's prefered location (see ``jupyter --paths`` command for more details):
```

Comment:
Small grammar corrections :-)

...will create a new configuration fo the kernel in one of the preferred locations...